pages/scan.tsx

```diff
@@ -198,10 +202,13 @@ Scan.getInitialProps = async ctx => {
   const profile = await getProfile(req);
 
   // Null profile means user is not logged in, and this is only relevant for admins
-  if (!profile || profile.role !== "admin") {
+  if (!profile || !(profile.role == "admin" || profile.role == "volunteer" || profile.role == "sponsor")) {
```

Comment:
Maybe just do `profile.role === 'hacker'`?

Reply:
Yeah could do that, I was thinking if we add more roles in the future like 'judge', they wouldn't need access to scan

Reply:
I'll change it and add a todo for better options